Sources/SwiftPieChart/PieChartView.swift

```diff
@@ -49,7 +56,7 @@ public struct PieChartView: View {
         GeometryReader { geometry in
             VStack{
                 ZStack{
-                    ForEach(0..<self.values.count){ i in
+                    ForEach(0..<self.values.count, id:\.self){ i in
```

Comment:
This is good because it will eliminate the compiler warning `Non-constant range: argument must be an integer literal`